### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by RXWE**
initial commit

### CODE REVIEW

**Comment by jangdonghae** [comment]
good job!

**Comment by dreamsh19** [comment]
Well done :)

File: frontend/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
```
Comment (deploy-soon) [inline frontend/package.json]: How about add some test options

Comment (RXWE) [reply frontend/package.json]: I will fix it soon

Comment (deploy-soon) [reply frontend/package.json]: Thanks please fix this before merge this branch

**Comment by deploy-soon** [comment]
Well done :)

**Pull request merged**